app/src/main/java/com/example/lapnguyen/weather_3/data/Weather.java

```diff
@@ -0,0 +1,87 @@
+package com.example.lapnguyen.weather_3.data;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * Created by lapnguyen on 15/06/2017.
+ */
+
+public class Weather {
+
+    private String currentLocation;
+
+    private String lastUpdateTime;
+
+    private String weatherIcon;
+
+    @SerializedName("temperature")
+    private double currentTemperature;
+
+    @SerializedName("windSpeed")
+    private double windSpeed;
+
+    @SerializedName("humidity")
+    private double humidityPercent;
+
+    public Weather() {
+    }
+
+    public Weather(String currentLocation, String lastUpdateTime, String weatherIcon, double currentTemperature, double winSpeed, double humidityPercent) {
+        this.currentLocation = currentLocation;
+        this.lastUpdateTime = lastUpdateTime;
+        this.weatherIcon = weatherIcon;
+        this.currentTemperature = currentTemperature;
+        this.windSpeed = windSpeed;
```

Comment:
ERROR: [CORRECTNESS][UR_UNINIT_READ] Uninitialized read of windSpeed in new com.example.lapnguyen.weather_3.data.Weather(String, String, String, double, double, double)
http://findbugs.sourceforge.net/bugDescriptions.html#UR_UNINIT_READ